### PR TITLE
Re-using children variable and default assigning it to Icon

### PR DESCRIPTION
**Pull request by mlunoe**
This PR is a follow-up PR to https://github.com/dcos/dcos-ui/pull/1822

### CODE REVIEW

**Comment by jfurrow** [comment]
:shipit:

File: src/js/components/ClipboardTrigger.js [hunk 1]
```diff
@@ -103,7 +106,11 @@ ClipboardTrigger.defaultProps = {
 
 ClipboardTrigger.propTypes = {
   children: PropTypes.node,
-  className: PropTypes.string,
+  className: PropTypes.oneOfType([
```
Comment (nLight) [inline src/js/components/ClipboardTrigger.js]: Looks like a recurring pattern, we could introduce a `propType` for that

Comment (jfurrow) [inline src/js/components/ClipboardTrigger.js]: JIRA: https://mesosphere.atlassian.net/browse/DCOS-13617

**Comment by mlunoe** [comment]
Closing because we are moving towards having the components handle their own classes

**Pull request closed without merging**